#! /bin/sh /usr/share/dpatch/dpatch-run
## 23_use_xdvi.bin.dpatch
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use xdvi.bin as xdvi binary, which is handled via alternatives
## DP: and not xdvi-xaw.real. The alternatives setting updates xdvi.bin
## DP: to xdvi-xaw.real

@DPATCH@
 texk/xdvik/xdvi-sh.in |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: texlive-bin-2009~svn13478/texk/xdvik/xdvi-sh.in
===================================================================
--- texlive-bin-2009~svn13478.orig/texk/xdvik/xdvi-sh.in	2009-05-26 10:59:48.000000000 +0200
+++ texlive-bin-2009~svn13478/texk/xdvik/xdvi-sh.in	2009-05-27 22:20:14.000000000 +0200
@@ -50,4 +50,4 @@
   XFILESEARCHPATH="$xdviapppath:${XFILESEARCHPATH-%D}"; export XFILESEARCHPATH
 fi
 
-exec @final_exec_name@ $NAMEOPT ${1+"$@"}
+exec xdvi.bin $NAMEOPT ${1+"$@"}
